README.md

```diff
@@ -267,9 +264,10 @@ Following that we will have to init the contract again with our own metadata. Th
 
 ```rust
 
+    use near_token::NearToken;
```

Comment:
Import from near_workspaces::types, so users don’t need to explicitly deal with the dependency